src/jsmind.js

```diff
@@ -449,6 +449,77 @@ export default class jsMind {
             return null;
         }
     }
+
+    /**
+     * Add multiple nodes to the mind map with optimized performance.
```

Comment:
I prefer to refactor `add_node`, extract 2 methods from it:
- _add_node_data(...)
- _refresh_node_ui(...)

then the method can be rewritten to :

```javascript

add_node() {
    this._add_node_data();
    this._refresh_node_ui();
}

add_nodes() {
    for(...) {
        this._add_node_data();
    }
    this._refresh_node_ui();
}

```

what do you think?